CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ All notable changes to this project will be documented in this file.
 - Clarified wording of pre-flight check messages (#457, #436)
 - Added caching of failure to find a docker daemon, so that subsequent tests fail fast. This is likely to be a significant improvement in situations where there is no docker daemon available, dramatically reducing run time and log output when further attempts to find the docker daemon cannot succeed.
 - Allowing JDBC containers' username, password and DB name to be customized (#400, #354)
+- Added support for explicitly setting file mode when copying file into container (#446) 
```

Comment:
As I've released 1.4.3 now, please could you shift up to the top of the changelog?

Reply:
So it will get to 1.4.4?

Reply:
Yes.